recipes/psrchive/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "psrchive" %}
+{% set version = "2021-01-18" %}
+
+package:
+    name: "{{ name | lower }}"
```

Comment:
@demorest this is minor but we've been using 2 spaces indentations for yaml for a while. That avoids long lines in multiple nested fields.

Reply:
OK that is easy enough to change.